c2corg_ui/static/js/feed.js

```diff
@@ -138,6 +336,60 @@ app.FeedController.prototype.getDocumentsFromFeed = function() {
   }.bind(this));
 };
 
+/**
+ * get latest topics
+ * @private
+ */
+app.FeedController.prototype.getLatestTopics_ = function() {
+  this.busyForum = true;
+  this.api.readLatestForum().then(function(response) {
+    this.handleForum(response);
+  }.bind(this), function() { // Error msg is shown in the api service
+    this.busyForum = false;
+    this.errorForum = true;
+  }.bind(this));
+};
+
+/**
+ * number cannot be < 0 and cannot be decimal
+ * @param {number} n
+ * @private
+ */
+app.FeedController.prototype.naturalNumber = function(n) {
```

Comment:
Is this function really used? I can see no other occurence of it in the code.
By the way:
* private function names must have a trailing _ (eg. naturalNumber_)
* prefer using a verb as name, not a noun (eg. getNaturalNumber_)
* the ``@return {number}`` jsdoc is missing

Reply:
fixed, i forget to remove it.